ui/pages/confirmations/components/fee-details-component/fee-details-component.js

```diff
@@ -27,42 +27,30 @@ import LoadingHeartBeat from '../../../../components/ui/loading-heartbeat';
 import UserPreferencedCurrencyDisplay from '../../../../components/app/user-preferenced-currency-display/user-preferenced-currency-display.component';
 import { PRIMARY, SECONDARY } from '../../../../helpers/constants/common';
 import { addHexes } from '../../../../../shared/modules/conversion.utils';
-import fetchEstimatedL1Fee from '../../../../helpers/utils/optimism/fetchEstimatedL1Fee';
 import { useGasFeeContext } from '../../../../contexts/gasFee';
 
 export default function FeeDetailsComponent({
   txData,
   useCurrencyRateCheck,
   hideGasDetails = false,
 }) {
+  const layer1GasFee = txData?.layer1GasFee;
   const [expandFeeDetails, setExpandFeeDetails] = useState(false);
-  const [estimatedL1Fees, setEstimatedL1Fees] = useState(null);
```

Comment:
I assume our empty value is now `undefined` if not set where as it was `null` before.

Should we use a `?? null` above for backwards compatibility?

Reply:
Done.